File: TBD/QL/src/ql/ast/type/QLString.java
package ql.ast.type;

import ql.ast.ASTNode;

/**
 * Created by Erik on 7-2-2017.
 */
public class QLString implements ASTNode {
    private String qlString;

    public QLString(String qlString){
        this.qlString = qlString;
    }
}

File: WeekendWarriors/Questionnaires/Questionnaires/AST/Types/QLString.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Questionnaires.AST
{
    public class QLString : INode
    {
        public QLString(string value)
        {
            this.Value = value;
        }

        public string Value
        {
            get;
        }
    }
}
